examples/src/bin/clone_macro.rs

```diff
@@ -1,8 +1,7 @@
 use std::cell::RefCell;
```

Comment:
Seems like we didn't add a module doc comment explaining what this example was about. :-/